californium-core/src/main/java/org/eclipse/californium/core/network/stack/BlockwiseLayer.java

```diff
@@ -630,6 +643,37 @@ public void receiveResponse(final Exchange exchange, final Response response) {
 				case REQUEST_ENTITY_INCOMPLETE: // 4.08
 					// we seem to have uploaded blocks not in expected order
 				case REQUEST_ENTITY_TOO_LARGE: // 4.13
+					if (enableAutoFailoverOn413) {
+						if (response.getOptions().hasBlock1()) {
+							// may be a block size negotiation
+							handleBlock1Response(exchange, response);
+							return;
+						} else if (!exchange.getRequest().isCanceled()
+								&& getBlock1Status(getKey(exchange, exchange.getCurrentRequest())) == null) {
```

Comment:
That's the casesf, which makes me feel: let's try, how we the request could be transmitted anyway. If you want it, do it.